From 6d3e64226d43a55f4554e1dcacaf56d81d3dae86 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Thu, 20 Nov 2014 14:40:19 -0500
Subject: Revert "screen: Set a black background for testing purposes"

This reverts commit ec8ed1dbb04ce8169c0a38ddf066d5565117c587.

1) It turns out to add a momentary flicker from the transition
between the login screen and user session
2) It actually isn't needed anymore since bug 733026

https://bugzilla.gnome.org/show_bug.cgi?id=740377

diff --git a/src/core/screen.c b/src/core/screen.c
index dbf3f60..b26137e 100644
--- a/src/core/screen.c
+++ b/src/core/screen.c
@@ -496,15 +496,6 @@ create_guard_window (Display *xdisplay, MetaScreen *screen)
   return guard_window;
 }
 
-/* Set a black background on the root window so that we don't
- * see confusing old copies of old windows when debugging
- * and testing. */
-static void
-meta_screen_set_background (MetaScreen *screen)
-{
-  XSetWindowBackground (screen->display->xdisplay, screen->xroot, 0x00000000);
-}
-
 MetaScreen*
 meta_screen_new (MetaDisplay *display,
                  int          number,
@@ -709,7 +700,6 @@ meta_screen_new (MetaDisplay *display,
   reload_monitor_infos (screen);
 
   meta_screen_set_cursor (screen, META_CURSOR_DEFAULT);
-  meta_screen_set_background (screen);
 
   /* Handle creating a no_focus_window for this screen */
   screen->no_focus_window =
-- 
cgit v0.10.1

